src/project-2.js

```diff
@@ -1,106 +1,96 @@
 // Do not change any of the function names
 
-const getBiggest = (x, y) => {
-  // x and y are integers.  Return the larger integer
-  // if they are the same return either one
-};
+const getBiggest = (x, y) => Math.max(x, y);
 
 const greeting = (language) => {
-  // return a greeting for three different languages:
-  // language: 'German' -> 'Guten Tag!'
-  // language: 'English' -> 'Hello!'
-  // language: 'Spanish' -> 'Hola!'
-  // if language is undefined return 'Hello!'
+  switch (language) {
+    case 'German':
+      return 'Guten Tag!';
+    case 'Spanish':
+      return 'Hola!';
+    default:
+      return 'Hello!';
+  }
 };
 
-const isTenOrFive = (num) => {
-  // return true if num is 10 or 5
-  // otherwise return false
-};
+const isTenOrFive = num => num === 5 || num === 10;
 
-const isInRange = (num) => {
-  // return true if num is less than 50 and greater than 20
-};
+const isInRange = num => num < 50 && num > 20;
 
-const isInteger = (num) => {
-  // return true if num is an integer
-  // 0.8 -> false
-  // 1 -> true
-  // -10 -> true
-  // otherwise return false
-  // hint: you can solve this using Math.floor
-};
+const isInteger = num => num % 1 === 0;
 
 const fizzBuzz = (num) => {
-  // if num is divisible by 3 return 'fizz'
-  // if num is divisible by 5 return 'buzz'
-  // if num is divisible by 3 & 5 return 'fizzbuzz'
-  // otherwise return num
+  const by3 = num % 3 === 0;
+  const by5 = num % 5 === 0;
+  const bitmap = [by5 && by3 ? 'fizzbuzz' : false, by5 === true ? 'buzz' : false, by3 === true ? 'fizz' : false];
+  let result = num;
+  bitmap.forEach((t, i) => {
+    result = t !== false && (result.length < t.length || result.length === undefined) ? t : result;
+  });
+  return result;
 };
 
 const isPrime = (num) => {
-  // return true if num is prime.
-  // otherwise return false
-  // hint: a prime number is only evenly divisible by itself and 1
-  // hint2: you can solve this using a for loop
-  // note: 0 and 1 are NOT considered prime numbers
+  for (let i = 2; i < num; i++) {
+    if (num % i !== 0) {
+      return true;
+    } return false;
+  } return false;
 };
 
 const returnFirst = (arr) => {
```

Comment:
You're returning the last item in the array here.
Also, naming a variable `l` could be confusing for other people to see.